### PR TITLE
Import bytecode library from Presto

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Slice
```
Comment (dain) [inline README.md]: Slice?

Comment (electrum) [reply README.md]: Fixed

File: src/test/java/io/airlift/bytecode/expression/BytecodeExpressionAssertions.java [hunk 2]
```diff
@@ -38,7 +39,7 @@
 
 public final class BytecodeExpressionAssertions
 {
-    private static final boolean DUMP_BYTECODE_TREE = false;
+    public static final AtomicBoolean DUMP_BYTECODE_TREE = new AtomicBoolean();
```
Comment (dain) [inline src/test/java/io/airlift/bytecode/expression/BytecodeExpressionAssertions.java]: The commit message says "Always dump tree in BytecodeExpressionAssertions", but the value is false by default.

Comment (electrum) [reply src/test/java/io/airlift/bytecode/expression/BytecodeExpressionAssertions.java]: Thanks, not sure what I was thinking when I wrote that. Will update.

**Pull request merged**